docs/en/introduction/comparison/juicefs_vs_s3fs.md

```diff
@@ -4,35 +4,35 @@ slug: /comparison/juicefs_vs_s3fs
+S3FS does not do special optimization for files. It acts like an access channel between local and object storage, making the local mount point see the same content as on the object storage browser, which makes it easy to use cloud storage locally. However, from another aspect, this simple architecture makes S3FS' retrieving, reading and writing of files require direct interaction with the object store, and network latency can impact strongly on performance and user experience.
 
-JuiceFS uses a technical architecture that separates data and metadata, where any file is first split into data blocks according to specific rules before being uploaded to the object store, and the corresponding metadata is stored in a separate database. The advantage of this is that retrieval of files and modification of metadata such as file names can be directly interacted with the more responsive database, bypassing the network latency impact of interacting with the object store.
+JuiceFS uses a technical architecture that separates data and metadata, where any file is first split into data blocks according to specific rules before being uploaded to the object storage, and the corresponding metadata is stored in a separated database. The advantage of this is that retrieval of files and modification of metadata such as file names can directly interact with the more responsive database, bypassing the network latency impact of interacting with the object store.
 
-In addition, when it comes to large files, although S3FS can solve the problem of transferring large files by uploading them in chunks, the nature of object storage dictates that appending files requires rewriting the entire object. For large files of tens or hundreds of gigabytes or even terabytes, repeated uploads are bound to waste a lot of time and bandwidth resources.
+In addition, when processing large files, although S3FS can solve the problem of transferring large files by uploading them in chunks, the nature of object storage dictates that appending files requires rewriting the entire object. For large files of tens or hundreds of gigabytes or even terabytes, repeated uploads waste a lot of time and bandwidth resources.
 
-JuiceFS avoids such problems by splitting individual files into chunks locally according to specific rules (default 4MiB) before uploading, regardless of their size. Any rewriting and appending of files eventually becomes the generation of new data blocks instead of modifying already generated data blocks, greatly reducing the waste of time and bandwidth resources.
+JuiceFS avoids such problems by splitting individual files into chunks locally according to specific rules (default 4MiB) before uploading, regardless of their size. Any rewriting and appending of files eventually becomes the generation of new data blocks instead of modifying already generated data blocks, which greatly reduces the waste of time and bandwidth resources.
```

Comment:
```suggestion
JuiceFS avoids such problems by splitting individual files into chunks locally according to specific rules (default 4MiB) before uploading, regardless of their size. Rewriting and appending any files eventually becomes the generation of new data blocks instead of modifying already generated data blocks, which greatly reduces the cost of time and bandwidth resources.
```

Reply:
```suggestion
JuiceFS avoids such problems by splitting individual files into chunks locally according to specific rules (default 4MiB) before uploading, regardless of their size. Rewriting and appending to any files eventually becomes the generation of new data blocks instead of modifying already generated data blocks, which greatly reduces the cost of time and bandwidth resources.
```
这里 eventually becomes the generation of new data blocks 的应该是 rewriting and appending 吧，介词改成 to 会不会好一些